src/cpu/gemm/f32/jit_avx_gemv_t_f32_kern.cpp

```diff
@@ -120,8 +120,7 @@ void jit_avx_gemv_t_f32_kern::innerloop(int unroll_m, int unroll_n) {
 // Outer loop.
 void jit_avx_gemv_t_f32_kern::outerloop(
         int unroll_x, int unroll_y, Label *&cur_outerloop_label) {
-    if ((unroll_x > M_UNROLL_) || (unroll_y > N_UNROLL_) || (unroll_y < 0)
```

Comment:
I think it is not a redundant check, but a typo: the second (or the first) check should read `unroll_x < 0`.

Reply:
@rsdubtso , thanks for the review and the clarification. Changes made.